libcaja-private/caja-file.c

```diff
@@ -3922,7 +3922,6 @@ caja_file_get_boolean_metadata (CajaFile *file,
 		result = FALSE;
 	} else {
 		g_error ("boolean metadata with value other than true or false");
```

Comment:
It is not obvious that the `else` never is reached. Anyway, this PR does not solve the issue, because it just lets the return value uninitialised...

Reply:
g_error calls to abort() for terminating the application, and it will result in a core dump, so the instruction below is never run

Reply:
Thanks, I never knew that.